massa-cipher/src/constants.rs

```diff
@@ -6,17 +6,14 @@
 
 use pbkdf2::Params;
 
-/// Cipher version
-pub const VERSION: u32 = 0;
-
 /// AES-GCM-SIV nonce size.
 ///
 /// A nonce is a single-use value which enables securely encrypting multiple messages under the same key.
 /// Nonces need not be random: a counter can be used so long as the values are never repeated under the same key.
 pub const NONCE_SIZE: usize = 12;
 
 /// `PBKDF2` salt size.
-pub const SALT_SIZE: usize = 12;
+pub const SALT_SIZE: usize = 16;
 
 /// `PBKDF2` hash parameters.
 pub const HASH_PARAMS: Params = Params {
```

Comment:
number of rounds does not follow spec: 600000

Reply:
Done